File: c#/WillWeMakeItToPump.cs
// determine whether we have enough fuel to make it to the pump

using System;

public class WillWeMakeItToPump
{
	public static bool ZeroFuel(uint distanceToPump, uint mpg, uint fuelLeft)
	{
		return mpg * fuelLeft >= distanceToPump;
	}

	public static void Main(string[] args)
	{
		Console.WriteLine(ZeroFuel(50, 25, 2)); // true
		Console.WriteLine(ZeroFuel(100, 50, 1)); // false
	}
}

File: py/will_we_make_it_to_pump.py
# determine if you will make it on gallons of gas or not


def zero_fuel(dist_to_pump, mpg, fuel_left):
    return fuel_left >= dist_to_pump / mpg


print(zero_fuel(3, 3, 1))  # true
print(zero_fuel(3, 2, 1))  # false
